manager/vulnerabilities_handler.py

```diff
@@ -88,16 +88,18 @@ def __init__(self, rh_account_id, cve_cache_from, cve_cache_keepalive, cves_with
                             ]
 
         # Use cache if not disabled + systems are not filtered + cache exists
+        is_cached = False
         if not DISABLE_ACCOUNT_CACHE and not is_not_cacheable_request(args) and cve_cache_from:
             count_subquery = self._cached_count_subquery(rh_account_id, cves_without_errata, args)
+            is_cached = True
```

Comment:
Nitpick: `is_cached` is not utilized in this commit.

Reply:
it is, it is sent to the apply filters function

Reply:
that is true, but no filter is utilizing it, therefore this change is unrelated :grin: